app/src/main/java/com/example/plantsapp/PlantsFragment.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.plantsapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.plantsapp.databinding.FragmentPlantsBinding
+
+class PlantsFragment : Fragment() {
+
+    private var _binding: FragmentPlantsBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentPlantsBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onDestroy() {
```

Comment:
You need to assign this variable to null inside of onDestroyView instead of onDestroy

https://developer.android.com/topic/libraries/view-binding